docs/gallery/External_Resources.py

```diff
@@ -37,24 +37,30 @@
 # as a string, but if you want to reference a key then you reference
 # the key object itself.
 
-data = Data(name="species", data='homo sapien')
-er.add_ref(container=data, field='Data', key='human', resource_name='NCBI_Taxonomy',
+data = Data(name="species", data=['Homo sapien', 'Mus Musculus'])
+er.add_ref(container=data, field='', key='Homo sapien', resource_name='NCBI_Taxonomy',
            resource_uri='https://www.ncbi.nlm.nih.gov/taxonomy', entity_id='NCBI:txid9606',
            entity_uri='https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=9606')
 
-# The data is stored in KeyTable, ObjectTable, EntityTable, and ResourceTable.
+er.add_ref(container=data, field='', key='Mus Musculus', resource_name='NCBI_Taxonomy',
+           resource_uri='https://www.ncbi.nlm.nih.gov/taxonomy', entity_id='NCBI:txid10090',
+           entity_uri='https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=10090')
 
-# This will return a pandas dataframe of the ResourceTable
-er.resources.to_dataframe()
+###############################################################################
```

Comment:
Can you also give an example where field is not ''? Here's an example:

```
# In the above example, field was empty because the data of the Data object was being associated with a resource. The Data object may also have attributes that you would like to
associate with resources. Similarly, a Container object may have several datasets and attributes that you would like to associate with resources. To disambiguate between these different fields, you can set the 'field' keyword argument corresponding to the name of the dataset or attribute containing the values being associated with a resource. For example:
genotypes = DynamicTable(name='genotypes', description='My genotypes')
genotypes.add_column('genotype_name')
genotypes.add_row(id=0, genotype_name='Rorb')
er.add_ref(container=genotypes, field='genotype_name', ...) # fill in from the example from the example spreadsheet
```